bin/cli.js

```diff
@@ -14,20 +14,28 @@ const program = new Command();
 
 program
   .version(pkg.version)
+  .option('--logLevel', 'change the log level of the function', 'warn')
+  .option('--port', 'change the port the runtime listens on', 8080)
   .arguments('<file>')
   .action(runServer);
 
 program.parse(process.argv);
 
 async function runServer(file) {
+  const programOpts = program.opts();
   try {
     let server;
+    let options = {
+      logLevel: programOpts.logLevel || process.env.FUNCTION_LOG_LEVEL,
+      port: programOpts.port || process.env.LISTEN_PORT
```

Comment:
```suggestion
      logLevel: programOpts.logLevel || process.env.LOG_LEVEL,
      port: programOpts.port || process.env.PORT
```
Maybe let's use better names. @zroubalik we don't have any specific env vars in `func` yet that I am aware of. What do you think of these?

Reply:
I think that at least some common prefix is helpful, so users could easily distinguish those specific env vars, so what about `FUNC_LOG_LEVEL` and `FUNC_PORT` ? Or something similar but with some prefix that we can standardize across runtimes.

Reply:
Works for me

Reply:
i'll update,  should those names also translate to the cli flags or are the names i used ok?

Reply:
Probably the CLI flags are OK. For the `func` binary, this is how we do it. (And now I realize what @zroubalik was referring to :laughing:).

https://github.com/boson-project/func/blob/ea0a75a7ccb6d00b8c859ff4cd311ad33fb8dbc3/cmd/build.go#L18